src/Interface/InterfacesList.js

```diff
@@ -128,7 +128,6 @@ const InterfacesList = ({ fetchInterfaces, fields, interfaces, orgId, stripes })
         formatter={resultsFormatter}
         rowFormatter={anchoredRowFormatter}
         visibleColumns={visibleColumns}
-        rowMetadata={orgId}
```

Comment:
I don't think it's used, but it produces a warning in browser console, so I've decided to remove it